### PR TITLE
[DOCFIX] Update Metastore Doc

### PR DESCRIPTION
**Pull request by bf8086**

### CODE REVIEW

File: docs/en/operation/Metastore.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ These tuning parameters primarily affect the behavior of the cache.
 
 The heap metastore is simple: it stores all metadata on the heap. This gives consistent,
 fast performance, but the required memory scales with the number of files in the
-filesystem. With 32GB of Alluxio master memory, Alluxio can support around 40 million files.
+filesystem. With 32GB of Alluxio master memory, Alluxio can support around 30 million files.
```
Comment (bf8086) [inline docs/en/operation/Metastore.md:59]: @calvinjia Based on the estimate of 1KB memory consumption per file, do you think we should update the capacity estimate here with a more realistic number?

Comment (calvinjia) [reply docs/en/operation/Metastore.md:59]: Yeah I think we can give the guidance of 2 GB = 1 million files and say 64 GB supports about 30 million files.

Comment (bf8086) [reply docs/en/operation/Metastore.md]: Thanks Calvin, updated.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**